### PR TITLE
Add include_tags to datadog_monitor

### LINKED ISSUE #57441: Support include_tags in datadog_monitor
##### SUMMARY
The Datadog monitor API supports an `include_tags` option, which is not yet supported by the `datadog_monitor` Ansible module.

> `include_tags` a Boolean indicating whether notifications from this monitor automatically inserts its triggering tags into the title. Default: True 

_https://docs.datadoghq.com/api/?lang=python#create-a-monitor_

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME
`datadog_monitor` module - [lib/ansible/modules/monitoring/datadog_monitor.py](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/monitoring/datadog_monitor.py)

##### ADDITIONAL INFORMATION
Supporting the `include_tags` Datadog monitor option in the `datadog_monitor` Ansible module will allow users to choose whether, or not, monitor tags are to be included in the titles of their triggered monitor notifications. Setting `include_tags` to `false` may make sense for organizations that have made the determination that tags in their notification title bloats things more than they add value.

```yaml
# Create a metric monitor that does not include tags in its notification title
- datadog_monitor:
    type: "metric alert"
    name: "Test monitor"
    state: "present"
    query: "datadog.agent.up.over('host:host1').last(2).count_by_status()"
    message: "Host [[host.name]] with IP [[host.ip]] is failing to report to datadog."
    tags:
      - "environment:test"
      - "owner:testy-mctesterson"
    include_tags: false
    api_key: "9775a026f1ca7d1c6c5af9d94d9595a4"
    app_key: "87ce4a24b5553d2e482ea8a8500e71b8ad4554ff"
```

### PR DESCRIPTION
**Pull request by twmartin**
##### SUMMARY

- Add `include_tags` to `datadog_monitor` module

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

- datadog_monitor

##### ADDITIONAL INFORMATION

- Closes #57441
- https://docs.datadoghq.com/api/?lang=python#create-a-monitor

### CODE REVIEW

File: lib/ansible/modules/monitoring/datadog_monitor.py [hunk 1]
```diff
@@ -132,6 +132,11 @@
           - Effective for sparse values.
         version_added: "2.7"
         type: str
+    include_tags:
+        description:
+          - Whether notifications from this monitor automatically inserts its triggering tags into the title.
+        version_added: "2.9"
+        type: bool
```
Comment (resmo) [inline lib/ansible/modules/monitoring/datadog_monitor.py:139]: please add the default 

~~~diff
         type: bool
+        default: yes
~~~

**Review by resmo — COMMENTED** [review]
needs_revision

File: lib/ansible/modules/monitoring/datadog_monitor.py (hunk 1 above)
Comment (twmartin) [reply lib/ansible/modules/monitoring/datadog_monitor.py:139]: Ah, yes. Thank you!

**Comment by twmartin** [comment]
Superseded by https://github.com/ansible-collections/community.general/pull/409

**Pull request closed without merging**